ocs_ci/ocs/resources/mcg.py

```diff
@@ -406,7 +406,17 @@ def request_aws_credentials(self):
         sleep(5)
 
         secret_ocp_obj = OCP(kind='secret', namespace=self.namespace)
-        cred_req_secret_dict = secret_ocp_obj.get(resource_name=creds_request.name, retry=5)
+        try:
+            cred_req_secret_dict = secret_ocp_obj.get(resource_name=creds_request.name, retry=5)
+        except CommandFailed:
+            logger.error(
+                'Failed to retrieve credentials request secret'
+
```

Comment:
unnecessary blank line

Reply:
It isn't strictly necessary, but I chose to put it there out of styling and readability choices